### PR TITLE
Added html for section 2

### LINKED ISSUE #3: html section 2
we need to create a layout for the section 2

### PR DESCRIPTION
**Pull request by Denis20101977**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,7 +31,17 @@ <h1 class="download__title">Peaceful shopping mindful money</h1>
       </div>
     </section>
     <!-- Секция 2 -->
-    <section>Секция 2</section>
+    <section class="warranty" id="warranty">
+      <div class="warranty__wrapper">
+        <div class="warranty__text-block">
+          <h2 class="warranty_title" warranty boost</h2>
+          <p class="warranty_copy">ZEN cards have a very neat feature that will help you whenever your electronic devices start failing after the standard warranty ends, which is pretty common.</p>
```
Comment (Palych18) [inline index.html:38]: Two spaces warranty__copy

File: index.html [hunk 2]
```diff
@@ -31,7 +31,17 @@ <h1 class="download__title">Peaceful shopping mindful money</h1>
...
+          <p class="warranty_copy">When you buy any electronics with your ZEN shopping Mastercard, their warranty gets extended by one year. How does it work? It simply does. Automatically.</p>
```
Comment (Palych18) [inline index.html:39]: Two spaces warranty__copy

**Review by ghost — COMMENTED** [review]
Fix

File: index.html [hunk 3]
```diff
@@ -31,7 +31,17 @@ <h1 class="download__title">Peaceful shopping mindful money</h1>
       </div>
     </section>
     <!-- Секция 2 -->
-    <section>Секция 2</section>
+    <section class="warranty" id="warranty">
+      <div class="warranty__wrapper">
+        <div class="warranty__text-block">
+          <h2 class="warranty_title" warranty boost</h2>
```
[the remark this answers is not in the document]
Comment (Denis20101977) [reply index.html]: fixed.

File: index.html (hunk 1 above)
Comment (Denis20101977) [reply index.html]: fixed.

File: index.html (hunk 2 above)
Comment (Denis20101977) [reply index.html]: fixed.

**Review by ghost — APPROVED** [review]
Good

**Pull request merged**